tsconfig.json

```diff
@@ -5,7 +5,7 @@
         "sourceMap": true,
         "target": "es6",
         "outDir": "build",
-        "baseUrl": "./packages"
+        "declaration": true,
```

Comment:
Are you sure the baseUrl isn't needed ?

Reply:
In fact, I don't know yet... I'm waiting for my next PR where I'll set up the build of the different workspace and npm publication with Lerna.
But for the moment, with `"baseUrl": "./packages"`, the build is done in a `build` folder at the project's root.